lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -49,13 +49,6 @@ class Cookbook < Base
           boolean:      true,
           default:      nil
 
-        option :delivery,
```

Comment:
We'd chatted before about leaving this around but making it a no-op. The point would be that any script or external tool that was written to run `chef generate cookbook -d` would not fail because we removed this. OTOH, it's only been in one release and it would be cruft we'd have to keep around for a bit. Thoughts?